sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -295,6 +295,70 @@ export class ApplicationOfferingChangeRequestService {
     });
   }
 
+  /**
+   * Get the Application Offering Change Request Status by the application id.
+   * @param applicationId the application id.
+   * @param studentId the student id.
+   * @returns application offering change request status.
+   */
+  async getApplicationOfferingChangeRequestStatus(
+    applicationId: number,
+    studentId: number,
+  ): Promise<ApplicationOfferingChangeRequestStatus> {
+    const applicationOfferingChangeRequest =
+      await this.applicationOfferingChangeRequestRepo.findOne({
+        select: {
+          id: true,
+          applicationOfferingChangeRequestStatus: true,
+          createdAt: true,
+        },
+        relations: {
+          application: {
+            student: true,
+          },
+        },
+        where: {
+          application: { id: applicationId, student: { id: studentId } },
+        },
+        order: {
+          createdAt: "DESC",
+        },
+      });
+    return applicationOfferingChangeRequest?.applicationOfferingChangeRequestStatus;
```

Comment:
As per the comment from @dheepak-aot (https://github.com/bcgov/SIMS/pull/2124/files#r1270971907) making the method more generic would be better, which would also include returning the object instead of the specific property, IMO.